### ISSUE TITLE
Unused recycler parts avalible.

### ISSUE THREAD

**Issue by DanStaal** [LouisB3/UKS-KPBS_Compatibility#12]
Digging through the parts, I found a couple of wedge recyclers that USI-LS could use, but don't currently have configs for them.  I have a first pass at a config in my github:
https://github.com/DanStaal/KSP-Configs/blob/master/PlanetaryBase_USI_LS.cfg  Feel free to use it if you want.

These are listed as a carbon extractor and a water purifier - My thought is that the carbon extractor does recycling for a lot of Kerbals, but doesn't extend your supplies that much.  (Since air is only a small part of the supplies.)   The water purifier does high-quality recycling, but can only handle the water for a couple of Kerbals.

**Comment by LouisB3** [comment]
Yup, those are for TAC-LS I believe, though they could be repurposed for USI-LS. 

I need to check out the latest KPBS update's USI-LS changes before I make any decisions about changing/tweaking my own configs.

**Comment by DanStaal** [comment]
No problem.  I'm just trying to learn Module Manager and how to write USI-LS config files, and these were two of my test cases.  I thought they might be useful, since they aren't covered already.  ;)
